Makefile

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+ifeq (${IMAGE_ORG}, )
+  IMAGE_ORG=openebs
+  export IMAGE_ORG
+endif
+
```

Comment:
needs to be set here , since the multi arch Makefile target excuted from the root Makefile